src/main/java/com/google/sps/server/ScheduleCalendar.java

```diff
@@ -70,7 +70,7 @@ public void addSchedule(Schedule schedule) throws IOException {
 
       // for each course in the schedule, add all section times to calendar
       for (ScheduledCourse currentCourse : schedule.getCourses()) {
-        List<TimeRange> lectureSectionTimes = currentCourse.getLectureSection().getMeetingTimes();
+        List<TimeRange> lectureSectionTimes = currentCourse.getlectureSection().getMeetingTimes();
```

Comment:
See comment in `Scheduler.java`, `getlectureSection` &rightarrow; `getLectureSection` (at least for the final pr into master)

Reply:
Thanks again. I'll find the correct commit that has these changes